src/sentry/models/minidump.py

```diff
@@ -0,0 +1,115 @@
+"""
+sentry.models.minidump
+~~~~~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2016 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+
+from __future__ import absolute_import
+
+from django.db import models, transaction
+from symbolic import ProcessState
+
+from sentry.constants import LOG_LEVELS_MAP
+from sentry.db.models import FlexibleForeignKey, Model, sane_repr
+from sentry.models.file import File
+
+
+class MinidumpFile(Model):
+    __core__ = False
+
+    file = FlexibleForeignKey('sentry.File')
+    event_id = models.CharField(max_length=36, unique=True)
```

Comment:
This needs `index=True`.

Reply:
Probably also a uniqueness constraint.

Reply:
`unique=True` should add a unique index, no?

Reply:
yep